scripts/_verify

```diff
@@ -18,11 +18,15 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-REPO_ROOT="${REPO_ROOT:-$(cd "$(dirname "$0")/.." && pwd)}"
+# NOTE: Usually you'd not run this directly.
```

Comment:
Usually you wouldn't run...

Reply:
This is a stylistic choice - I'd say "you'd not run" like this in speech, personally, and I think it's understandable; thanks for the suggestion though!